openc3/lib/openc3/models/gem_model.rb

```diff
@@ -90,30 +80,33 @@ def self.install(name_or_path, scope:)
         Gem.done_installing_hooks.clear
         Gem.install(gem_file_path, "> 0.pre", :build_args => ['--no-document'], :prerelease => true)
```

Comment:
What are these options doing: "> 0.pre" and prerelease

Reply:
This allows it to install beta gems like we have during development.